src/components/Carousel/Carousel.jsx

```diff
@@ -0,0 +1,239 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+import Button from '../Button/index';
+import ArrowRight from '../Icon/Icons/ArrowRight';
+import ArrowLeft from '../Icon/Icons/ArrowLeft';
+import { gray, white } from '../style/colors';
+
+const CarouselStyled = styled.div`
+  align-items: center;
+  display: flex;
+`;
+
+const Window = styled.div`
+  overflow: hidden;
+  width: ${props => props.width};
+  margin: 8px;
+  display: flex;
+`;
+
+const ButtonOverlapContainer = styled.div`
+  position: relative;
+  z-index: 3;
+
+  ${props =>
+    props.left
+      ? css`
+          left: 20px;
+        `
+      : css`
+          right: 20px;
+        `}
+`;
+
+const MainList = styled.ul`
+  display: flex;
+  padding: 0;
+  margin: 0;
+  position: relative;
+  left: ${props => `${props.left}px`};
+  transition: left 0.5s ease-in;
+`;
+
+const MainListItem = styled.li`
+  list-style: none;
+`;
+
+// grabbed these styles from https://a11yproject.com/posts/how-to-hide-content/
+const Announcement = styled.div`
+  position: absolute !important;
+  height: 1px;
+  width: 1px;
+  overflow: hidden;
+  clip: rect(1px 1px 1px 1px); /* IE6, IE7 */
```

Comment:
@gmzjuliana do we need to be supporting ie6/7? I don't think that event Microsoft supports them 🤔

Reply:
Yep, no need for that.